src/MusicTheory/PitchClass.elm

```diff
@@ -113,27 +94,6 @@ all =
         |> List.concatMap (\l -> accidentals |> List.map (pitchClass l))
```

Comment:
What's a use case for `PitchClass.all`?

Reply:
I need that for Luigi, e.g. Again not necessary, just convenience.